packages/mobile/src/config.ts

```diff
@@ -52,6 +52,8 @@ export const ESCROW_PAYMENT_EXPIRY_SECONDS = 86400 // 1 days
 export const DEFAULT_TESTNET = Config.DEFAULT_TESTNET || 'integration'
 export const DAILY_PAYMENT_LIMIT_CUSD = 500
 export const SMS_RETRIEVER_APP_SIGNATURE = Config.SMS_RETRIEVER_APP_SIGNATURE
+// ODIS minimum dollar balance for pepper quota retrieval
+export const ODIS_MINIMUM_DOLLAR_BALANCE = 0.1
```

Comment:
Please leave a TODO to change this to 0.01 and add the CELO min balance one signer update goes out

Reply:
Is it possible to have this not require a re-release once the change is made within attestation service